### PR TITLE
Additional SSO Fault codes

### LINKED ISSUE #361: Additional SSO Fault codes
Found this on Ferroamp facebook group. Not sure where it came from originally though

image

### PR DESCRIPTION
**Pull request by bj00rn**
fixes #361

### CODE REVIEW

**Comment by argoyle** [comment]
You would need to rewrite the commit-message to conform to the conventional commits format (not sure if rewriting is possible via the web-UI). There's also a failing test where the expected value needs to be changed.

**Comment by argoyle** [comment]
At the bottom of the README.md there is a short section on contributing which might be a good read.

Looks good apart from those comments. 🎖️

**Comment by bj00rn** [comment]
> At the bottom of the README.md there is a short section on contributing which might be a good read.
> 
> Looks good apart from those comments. 🎖️

TL;DR :-) 

Should be ok now

**Review by argoyle — APPROVED** [review]
💯🚀

**Comment by argoyle** [comment]
Commit-lint is a bitch sometimes though. 😁 You need to start the message with a lower case letter. So one more rewrite and we're home.

**Comment by bj00rn** [comment]
> Commit-lint is a bitch sometimes though. 😁 You need to start the message with a lower case letter. So one more rewrite and we're home.

FFS... :-)  amended now

**Comment by bj00rn** [comment]
@argoyle added fault code 4 too
